### PR TITLE
Add needles for Krypton as of 2016-07-31

### PR DESCRIPTION
**Pull request by okurz**

### CODE REVIEW

File: generic-desktop-kde-live-20160731.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "tags": [
+    "generic-desktop"
```
Comment (coolo) [inline generic-desktop-kde-live-20160731.json]: Shouldn't this be limited to FLAVOR-Krypton as well?

Comment (okurz) [inline generic-desktop-kde-live-20160731.json]: I don't think so. I missed ENV-DESKTOP-kde however. Limiting to Krypton would be counter productive as we can probably expect any plasma related needle in Krypton to hit TW about 2-6 weeks later or not?

**Comment by okurz** [comment]
added boot menu needle

File: generic-desktop-kde-live-20160731.json (hunk 1 above)
Comment (coolo) [inline generic-desktop-kde-live-20160731.json]: but not Leap - and you match TW branding also on leap. we already have quite some generic-desktop needles ;(

Comment (okurz) [inline generic-desktop-kde-live-20160731.json]: hm, ok, so ENV-LEAP-0. It should still match FLAVOR-Argon, though, right?

Comment (okurz) [inline generic-desktop-kde-live-20160731.json]: maybe you can help with https://progress.opensuse.org/issues/12956#open-questions

**Comment by sysrich** [comment]
the generic-desktop needle seems suspicious - I'd expect a KDE live desktop to have a Desktop Folder, like https://openqa.opensuse.org/tests/232774#step/first_boot/1 - even on Krypton

**Comment by okurz** [comment]
removed the desktop needle without icons

**Pull request merged**